File: OCP/ Kata_ Shops Types/SolucionPhp/Shop/Shop/Type/ToyStore.php
<?php

declare(strict_types=1);

namespace SolucionPhp\Shop\Shop\Type;

class ToyStore implements ToyShopInterface
{
    private const HUMAN_TYPE = 'ToyStore';
    private const PLAYPLACE_AVAILABLE = true;

    public function getType(): string {
        return self::HUMAN_TYPE;
    }

    public function playPlaceAvailable(): bool
    {
        return self::PLAYPLACE_AVAILABLE;
    }

}

File: OCP/ Kata_ Shops Types/SolucionJava/Shop/Shop/Type/ToyStore.java
package SolucionJava.Shop.Shop.Type;

public class ToyStore implements ToyShopInterface {
    private final Boolean PLAY_PLACE = true;
    private final String HUMAN_TYPE_NAME = "Toy store";

    public String getType() {
        return HUMAN_TYPE_NAME;
    }

    public Boolean playPlaceAvailable() {
        return PLAY_PLACE;
    }
}
